contracts/MultiPartyEscrow.sol

```diff
@@ -0,0 +1,272 @@
+pragma solidity ^0.4.24;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+
+contract MultiPartyEscrow {
+    
+    //it seems we don't need SafeMath
```

Comment:
Yes, we need. When using the SafeMath library, an error will be thrown stopping the execution of the contract and it conforms to the existent codebase.

Reply:
Using SafeMath for MultiPartyEscrow is a big question. I will open an Issue and explain why.  And I don't understand your last part. If there is an error in the contract it must revert, there is not other possible behavior.... Ok we continue in an issue

Reply:
It should'nt be matter of discussion. Better some `wei` more than an overflow bug in mainnet.

It will be the first contract of the codebase not using it. Strange.

Reply:
What do you mean "codebase"? If it is signet/platform-contract then I am the first who consider using it (please make "grep SafeMath contracts/*"). And please use issue https://github.com/singnet/platform-contracts/issues/45 for this discussion...

Reply:
And Marco have you read what I've wrote in #45 ? That we are already checking for it? Etc... Too much discussions...